Changing the script naming for the with-reason-react example

Hi,

When creating a new project with `reason-react` the following guide appears:
```
To start a local server for development:
  yarn start
```
but as shown inside the `reason-react` example, `yarn run dev` and not `yarn start` is required to run the example.

If you just run `yarn start`, you get a cryptic error message that doesn't really help:
```
❯ npm start

> razzle-examples-reason-react@0.7.2 start 
> NODE_ENV=production node build/server.js

module.js:487
    throw err;
    ^

Error: Cannot find module 'build/server.js'
```

Either changing the post-installation message or changing the scripts required to run the reason-react example would help beginners get started. I'd be happy to PR this if needed.

--- a/examples/with-reason-react/package.json
+++ b/examples/with-reason-react/package.json
@@ -3,13 +3,13 @@
   "version": "0.7.2",
   "license": "MIT",
   "scripts": {
-    "dev": "npm run js-watch | npm run bsb-watch",
+    "start": "npm run js-watch | npm run bsb-watch",
     "js-watch": "razzle start",
     "bsb-watch": "bsb -make-world -w",
     "clean": "bsb -clean-world && rm -rf build",
     "build": "bsb -make-world && razzle build",
     "test": "razzle test --env=jsdom",
-    "start": "NODE_ENV=production node build/server.js"
+    "start:prod": "NODE_ENV=production node build/server.js"
   },
   "dependencies": {
     "express": "^4.15.2",

